README.md

```diff
@@ -4,82 +4,58 @@
 
 Create a JS/TypeScript library using our Rest API to show content of top of an HTML 5 Video Player
 
+## Requirements:
 
-
-## Part A: HTML Player
-
-Create a simple HTML page, that hosts a video stream (**use this URL**: https://buffup-public.s3.eu-west-2.amazonaws.com/video/toronto+nba+cut+3.mp4) in any HTML5 video player you want.
-
-Import the npm package for the library that you have created in Part B, to display content on top of that player
-
-## Part B: Web Library
-
-We now want a JS/Typescript library distributed as an npm package, that allows to display content (Buff's as we call them) over any existing video player.
-
-The Developer using our library should import our library and initialize it with the ID of the player.
-
-The library then should locate the HTML player frame, and render on top of that HTML content (our Buffs)
-
-### Library Requirements
+We now want a JS/Typescript file that allows to display content (Buff's as we call them) over any existing video player.
```

Comment:
Only TypeScript

Reply:
Turned repo into a small app to support TS compilation